app/assets/stylesheets/styles/components/guides/components/_guide_creation_nav.css.scss

```diff
@@ -0,0 +1,84 @@
+.guide-creation-nav {
+  $guide-nav-color: #5E9546;
+  background: $guide-nav-color;
+
+  /* the ul containing the links */
+  ul {
+    padding-top: 5px;
+    text-align: center;
+    list-style: none;
+    white-space: nowrap;
+  }
+
+  li {
+    position: relative;
+    display: inline-block;
+  }
+
+  li a {
+    color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.